zebra-chain/src/transaction.rs

```diff
@@ -343,6 +343,13 @@ impl Transaction {
         }
     }
 
+    /// Access the [`transparent::OutPoint`]s spent by this transaction.
```

Comment:
Doc nit: `transparent::Output`s can also be represented by `OutPoint`s, and they're easily confused.

So it might help to mention inputs as well:
```suggestion
    /// Access the [`transparent::OutPoint`]s spent by this transaction's [`transparent::Input`]s.
```